language/test/Language_test.rb

```diff
@@ -162,6 +164,86 @@
     end
   end
 
+  describe "#Save" do
+    let(:language) { "es_ES" }
+    let(:initial_stage) { false }
+    let(:command_result) { { "exit" => 0 } }
+
+    before do
+        allow(Yast::Stage).to receive(:initial).and_return(initial_stage)
+
+        allow(Yast::WFM).to receive(:Execute).and_return(command_result)
+        allow(Yast::SCR).to receive(:Execute).and_return(command_result)
+        allow(Yast::SCR).to receive(:Write)
+        allow(subject).to receive(:valid_language?).with(language).and_return(true)
+
+        subject.Set(language)
+        subject.SetDefault
+    end
+
+    it "writes .sysconfig.language.INSTALLED_LANGUAGES" do
+      expect(Yast::SCR).to receive(:Write).with(Yast.path(".sysconfig.language.INSTALLED_LANGUAGES"), anything)
+
+      subject.Save
+    end
+
+    it "cleans .sysconfig.language in sysconfig" do
```

Comment:
this is wrong. It does not mean cleaning. It is pushing cache. If you write to file path in SCR nil, it means flush cache and write immediately.